### PR TITLE
Chrysler updates 🚐🚐🚐

### PR DESCRIPTION
**Pull request by adhintz**
- use can packer
- more closely emulate stock LKAS messages
- tests for chryslercan
- ~~ramp down steer angle on disengage to avoid fault~~

This relies on this OpenDBC PR: https://github.com/commaai/opendbc/pull/140

### CODE REVIEW

File: selfdrive/car/chrysler/carcontroller.py [hunk 1]
```diff
@@ -39,22 +40,19 @@ def __init__(self, dbc_name, car_fingerprint, enable_camera):
...
-                                                   CS.steer_torque_motor, SteerLimitParams)
-
     moving_fast = CS.v_ego > CS.CP.minSteerSpeed  # for status message
     lkas_active = moving_fast and enabled
-
     if not lkas_active:
       apply_steer = 0
-
+    apply_steer = apply_toyota_steer_torque_limits(apply_steer, self.apply_steer_last,
+                                                   CS.steer_torque_motor, SteerLimitParams)
```
Comment (rbiasini) [inline selfdrive/car/chrysler/carcontroller.py]: Mmm... I think panda does not let any non-zero going through immediately after disengagement

**Comment by adhintz** [comment]
You're right. I also need to send a panda PR.

Was going to send one after https://github.com/commaai/panda/pull/151
but I can make a single larger Panda PR if you'd prefer.

**Comment by rbiasini** [comment]
@adhintz commaai/panda#151 is merged. You sure that you have faults if you put the rate limiter before the lkas_active check?. I remember spending a bit of time on this and concluded it wasn't an issue.

**Comment by rbiasini** [comment]
TODOs:
1- https://github.com/commaai/opendbc/commit/ecf99e038c4518341486157b2f6980ac7fff8491 needs to be in this PR 
2- update README with new car supports
3- update panda with modified safety code

We can make a patch for 1 and 2.

**Comment by adhintz** [comment]
You're right. I reverted the change so it will now go down to 0 instantly. It worked fine on three drives this evening and didn't fault anything.

> @adhintz [commaai/panda#151](https://github.com/commaai/panda/pull/151) is merged. You sure that you have faults if you put the rate limiter before the lkas_active check?. I remember spending a bit of time on this and concluded it wasn't an issue.

**Comment by legonigel** [comment]
Merged in #537. Thanks!

**Pull request closed without merging**